tests/installation/scc_registration.pm

```diff
@@ -59,6 +59,16 @@ sub run {
                 bugref       => 'bsc#1028774'
             );
         }
+        # we need to confirm 'Yes' for the keep registered scenario,
+        # which migrate from sle12sp4 to sle12sp5.
+        if (match_has_tag('registration-online-repos')) {
+            if (is_sle('=12-SP5') && check_var('KEEP_REGISTERED', '1')) {
+                wait_screen_change { send_key('alt-y') };
+                assert_screen('module-selection');
+                wait_screen_change { send_key('alt-n') };
+                return;
```

Comment:
Could we change to return like ' return wait_screen_change { send_key('alt-n') };' ?